Sources/SpeziBluetooth/AccessorySetupKit/AccessorySetupKit.swift

```diff
@@ -0,0 +1,381 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+#if canImport(AccessorySetupKit) && !os(macOS)
+import AccessorySetupKit
+#endif
+import Foundation
+import Spezi
+
+
+/// Enable privacy-preserving discovery and configuration of accessories through Apple's AccessorySetupKit.
+///
+/// This module enables to discover and configure Bluetooth or Wi-Fi accessories using Apple's [AccessorySetupKit](https://developer.apple.com/documentation/accessorysetupkit).
+///
+/// - Important: Make sure to follow all the setup instructions in [Declare your app's accessories](https://developer.apple.com/documentation/accessorysetupkit/discovering-and-configuring-accessories#Declare-your-apps-accessories)
+///     to declare all the necessary accessory information in your `Info.plist` file.
+///
+/// ## Topics
+///
+/// ### Configuration
+/// - ``init()``
+///
+/// ### Discovered Accessories
+/// - ``accessories``
+///
+/// ### Observe Accessory Changes
+/// - ``AccessoryEvent``
+/// - ``AccessoryChangeHandler``
+///
+/// ### Displaying an accessory picker
+/// - ``showPicker(for:)``
+/// - ``pickerPresented``
+///
+/// ### Managing Accessories
+/// - ``renameAccessory(_:options:)``
+/// - ``removeAccessory(_:)``
+///
+/// ### Managing Authorization
+/// - ``finishAuthorization(for:settings:)``
+/// - ``failAuthorization(for:)``
+///
+/// ### Determine Support
+/// - ``supportedProtocols``
+/// - ``SupportedProtocol``
+@MainActor // TODO: should this SpeziBluetooth actor?
```

Comment:
I am wondering how we generally manage he `@MainActor`/`@BluetoothActor` issue with Swift 6? It seems like `@MainActor` is the way to go for any `@Observable` elements and therefore also here?

Reply:
We resolved this in #51